Mathlib/NumberTheory/Modular.lean

```diff
@@ -435,7 +435,8 @@ theorem eq_zero_of_mem_fdo_of_T_zpow_mem_fdo {n : ℤ} (hz : z ∈ 𝒟ᵒ) (hg
     _ = 1 := add_halves 1
 #align modular_group.eq_zero_of_mem_fdo_of_T_zpow_mem_fdo ModularGroup.eq_zero_of_mem_fdo_of_T_zpow_mem_fdo
 
-/-- Any `z : ℍ` can be moved to `𝒟` by an element of `SL(2,ℤ)`  -/
+/-- **First Fundamental Domain Lemma**: Any `z : ℍ` can be moved to `𝒟` by an element of
```

Comment:
I'm not sure this is really a named theorem is it?

Reply:
OK I reverted that